pkg/kapp/cmd/tools/diff_flags.go

```diff
@@ -38,4 +39,5 @@ func (s *DiffFlags) SetWithPrefix(prefix string, cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&s.AgainstLastApplied, prefix+"against-last-applied", true, "Show changes against last applied copy when possible")
 
 	cmd.Flags().StringVar(&s.Filter, prefix+"filter", "", `Set changes filter (example: {"and":[{"ops":["update"]},{"existingResource":{"kinds":["Deployment"]}]})`)
+	cmd.Flags().BoolVar(&s.ChangeYaml, prefix+"apply-run", false, "Print YAML to be applied and exit.")
```

Comment:
ChangeYaml -> ChangeYAML
Also ChangeYAML along with diff-aply-run seems a bit weird.
`Print YAML to be applied and exit.` -> `Print YAML to be applied`

Reply:
Will change diff-apply-run to something else, discussion is still on for the flag name.

Reply:
Updated